src/schema/v2/artwork/index.ts

```diff
@@ -334,6 +335,36 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           return normalizeImageData(size ? _.take(sorted, size) : sorted)
         },
       },
+      inquiryQuestions: {
+        type: new GraphQLList(InquiryQuestionType),
+        description:
+          "Structured questions a collector can inquire on about this work",
+        resolve: ({ ecommerce, inquireable }) => {
+          if (!ecommerce && inquireable) {
+            /**
+             * TODO: This is temporary, fixture data hardcoded to enable the
+             * first inquiry project to be broken down into parallel tasks. Will
+             * be followed w/ a PR to hook it up to a soon to be built gravity
+             * hook.
+             */
+            return [
+              {
+                id: "price_and_availability",
+                question: "Price & Availability",
```

Comment:
Nice! I can imagine we can extend it to include something like a type so that UI knows it's going to render a checkbox, etc.